### DISCUSSION TITLE
DaffodilModule

### DISCUSSION THREAD

**Discussion by LucasHiago** [graycoreio/daffodil#1873] [Q&A] [answered]
hi, how i import DaffodilModule.forRoot({BASE_URL: "YOUR_STORE_URL_GOES_HERE"}) ???

**Comment by damienwebdev** [comment]
@LucasHiago I'm fairly certain that whatever doc you're referencing here is really old (mea culpa).

If you give a little context of what you're trying to do, I can help point you in the right direction.

**Reply by LucasHiago** [reply]
I try follow the steps on INSTALATION.md

> Be sure to replace the default `YOUR_STORE_URL_GOES_HERE` with your store's url in your `AppModule`.

@NgModule({
    ...
    imports: [
        ...
        StoreModule.forRoot({}),
        EffectsModule.forRoot([]),
        DaffodilModule.forRoot({BASE_URL: "YOUR_STORE_URL_GOES_HERE"}),  => this don't have a importer. 
        ...
    ],
    ...
})
export class AppModule { }

**Reply by griest024** [reply]
`DaffodilModule` is not a real import, its just a placeholder for the specific package module e.g. `DaffCartStateModule` out of `@daffodil/cart/state`. You should install all the packages that you need and import the modules that correspond to the features that you require for your app.

The individual libs themselves probably have better docs: https://github.com/graycoreio/daffodil/blob/develop/libs/cart/guides/installation.md, etc.

**Reply by LucasHiago** [reply]
Thank you guys!

**Comment by damienwebdev** [comment, marked as answer]
@LucasHiago I think it's probably best if you describe the exact feature you're trying to accomplish. It's probably fairly unclear from the docs that Daffodil is a package framework where you can really pick and choose exactly what you need. 

It does, by now, pretty much support all the typical E-commerce behaviors, but we intentionally don't give you direction to use "all the packages" which can make getting started a little difficult.

**Reply by damienwebdev** [reply]
A final comment here @LucasHiago: I have created an issue to make this entrypoint a little smoother. If you have any commentary, please feel free to [add to the issue.](https://github.com/graycoreio/daffodil/issues/1876)
